test/integration/content-preview/ContentPreview.e2e.test.js

```diff
@@ -1,14 +1,53 @@
 // <reference types="Cypress" />
+const COLLECTION = [
+    Cypress.env('FILE_ID_DOC_VERSIONED'),
+    Cypress.env('FILE_ID_DOC'),
+    Cypress.env('FILE_ID_SKILLS'),
+    Cypress.env('FILE_ID_VIDEO'),
+];
 
-describe('ContentExplorer', () => {
-    beforeEach(() => {
-        cy.visit('/Elements/ContentExplorer');
-        cy.contains('An Ordered Folder').click();
-    });
+describe('ContentPreview', () => {
+    const helpers = {
+        load({ features, fileId } = {}) {
+            cy.visit('/Elements/ContentPreview', {
+                onBeforeLoad: contentWindow => {
+                    contentWindow.FEATURES = features;
+                    contentWindow.FILE_ID = fileId;
+                    contentWindow.PROPS = {
+                        collection: COLLECTION,
+                    };
+                },
+            });
+        },
+        checkPreviewHeader() {
+            cy.getByTestId('previewheader')
+                .as('previewheader')
+                .contains('Document - Versioned (PDF).pdf');
```

Comment:
We may want to check that `bcpr-header--basic` is not present, instead, in case someone uploads a new version/title to this test file on accident.